sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -51,7 +51,7 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-sdk-bom</artifactId>
-        <version>1.2.6</version>   <!-- NOTE: This should be updated manually. -->
```

Comment:
Still not correct. The latest version from maven central is [1.2.8](https://mvnrepository.com/artifact/com.azure/azure-sdk-bom). Could you help update the version and compare the `azure-core` version used in spring-cloud-azure-core and the one in `azure-sdk-bom`, to see whether they are the same?

Reply:
`azure-core` in `spring-cloud-azure-core` is 1.34.0 and in `azure-sdk-bom` is 1.33.0